querydsl-sql/src/test/java/com/querydsl/sql/SelectBase.java

```diff
@@ -395,7 +395,7 @@ public void Custom_Projection() {
     }
 
     @Test
-    @IncludeIn({H2, SQLSERVER, MYSQL, ORACLE, TERADATA}) // TODO fix postgresql
+    @ExcludeIn({CUBRID, DB2, DERBY, HSQLDB, POSTGRESQL, SQLITE, TERADATA})
```

Comment:
Still impossible to cherry pick, but it's okay.
Maybe it's a good idea to always order it alphabetically? (I don't know if there is a checkstyle check for that?)